cmd/libs/go2idl/make-client/generators/imported_types.go

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I would like to avoid having this file - see my comment above where you are using these vars.